### ISSUE TITLE
Write Occultish Newsletter / call to action for artists 

### ISSUE THREAD

**Issue by DanM3rcurius** [SecretDecks/Contentwriting#18]
# SecretDecks® Newsletter to Occultish **(text edits from Nathalie J.)**
## Intro
Calling on all Creators! SecretDecks looks to you if you’ve ever reinterpreted the Tarot or perhaps feel inspired to do so? 
We want to allow you a chance to showcase your recreating of the deck and sell your Tarot art as a digital collectible in a new innovative way. A way we believe not only has huge momentum now, but also builds on a technology that is rocketing into the future while also allowing art and its creators more autonomy. ((More expanded on why its the ‘way” of the future in a non technical way))

SecretDecks® builds on blockchain technology with an autonomous organisation. Your art will be conjured into a cryptographically secured token called an NFT. 
The copyright will always remain with you the creator(s).

###What is a digital collectable / NFT ?
For the ease of explanation - a comparison between a virtual collectible versus a physical collectible (like a painting ) is that each NFT contains unique information (metadata), like the creators digital fingerprint and the certificate of authenticity. That makes it both distinct from any other NFT and easily verifiable.
This makes the creation and circulation of fake collectibles distinctively harder as all artwork can be verified and traced back to the original creator. Similarly to a painting it will be a personal exchange between buyer and artist. And the rules followed accordingly.

### How does and NFT built on the blockchain differ from simply selling it from a website on the Internet? 
On no website can you sell digital unique art pieces also without the need to setup a payment gateway.  Additionally, sales as NFT enable life long royalties.

### I am not a digital artist and/or don’t use the computer for my works? 
No worries, we connect you with creative minds that are technically well versed, in a remote incubator program! You can find collaborators to create new works or digitize already existing ones! 
How do I get around this ? see above!

## *—— what we could also say in the end so make ppl less shy to ask questions should we miss any*

We seek to create an autonomous organisation for creators. From creators for creators. Hence we'll create and lay the groundwork in the same manner.  So please feel free to ask any question that you may have. 

We are well experienced with the technical innovations of NFT’s and Crypto in general. It is exactly why we look to you for the potential marriage  of these two “on the offset’ different realities (Crypto & Tarot). We want to combine these different fields of expertise as we essentially only want to make it more private and self organising 

#### What and Who is SecretDecks®?
Creators that seek to introduce the Tarot, in way of divination and communication with Your subconscious, to a new realm of self-sovereignty. SecretDecks will function as a soul like catalyst for an Autonomous Creator Organisation (ACO).

#### What is an Autonomous Creator Organisation?
An organisational layer (network) for groups of creators clustered around an idea. Within this network they can autonomously coordinate curation, voting (consensus building), funding, creation, as well as distribution and sales of digital assets. Supported by a token (that grows in value over time, a s network grows and tokens are bought and burned).

#### How do they come together?
Digital collectibles in the form of Tarot cards. Conjured up by creators across the globe and sold within an ecosystem of SecretDecks® apps that work together to fuel demand for the collectible cards.
In order to generate interest and value as well as motivate participants and test the ACO concept in a real world environment, it is required to have some value adding mechanism. Thus a small percentage (aopr. 0.5%) of the collectible sales will go to the ACO network. 
The concept is working if at least 85% of sale revenue is received by the creator(s) directly and more than 12 drops of decks have been successfully facilitated.

**Comment by evolayak** [comment]
Created a new file in the main repo with my edits and additons: [SecretDecks® Newsletter to Occultish] (https://github.com/SecretDecks/Contentwriting/blob/97aa31e2e92339a2f844c4d590790991dc609e9b/SecretDecks%C2%AE%20Newsletter%20to%20Occultish)

I'm playing around and trying to figure out all these Git processes, so bear with me!

**Comment by evolayak** [comment]
Created a new file in the main repo with my edits and additons: [SecretDecks® Newsletter to Occultish](https://github.com/SecretDecks/Contentwriting/blob/97aa31e2e92339a2f844c4d590790991dc609e9b/SecretDecks%C2%AE%20Newsletter%20to%20Occultish)

I'm playing around and trying to figure out all these Git processes, so bear with me!

**Comment by DanM3rcurius** [comment]
Content is done, what's next? Should we blast it out without styling? @wlstylr Do you remember what mail server they used?

**Comment by tamagit80** [comment]
Morning, do we have all the emails? We could create a list and use the
newsletter system. There is also the option html or plain text. I can send
you the login, let me know. Talk later g.

**Comment by tamagit80** [comment]
Would be interesting to see how many read it or cancel the subscription
ecc... to get some info out of it. Newsletter can be connected to
analytics.

**Comment by DanM3rcurius** [comment]
> Morning, do we have all the emails? We could create a list and use the newsletter system. There is also the option html or plain text. I can send you the login, let me know. Talk later g.

Morning! The issue is referencing the Occultish Newsletter, to which we don't have the mails. Nat's friend will push it via their mail client; think they said its HTML.

Regarding our Newsletter: Robert (who listened in on our TownHall yesterday) is a beautiful mind (his specialty is NeurolinguisticProgramming!) offered his time and attention to take care of the newsletter server (management, sending, debugging, maintenance). if possible please connect to him via Discord and give him the login - or should he set up a new one and we transfer? feel free to discuss this with him and when you come to a conclusion please pin it in an issue in guthub 🙏🏿

**Comment by wlstylr** [comment]
> Content is done, what's next? Should we blast it out without styling? @wlstylr Do you remember what mail server they used?

I forgot the exact name of the program that they use, but they are owned by Mailchimp; basically it is a stripped down version of Mailchimp.  

HTML or plain text should work.  Probably best to see what format Occultish prefers. 

Before we send out I think we should add categories and channels to the discord server that will speak to Tarot, NFT, and artist communities.  Current discord is structured to support a DisCO and get work done.  We also need to have content that gets people excited about art and Tarot that creates excitement and encourages interaction.  Maybe something like "Get a live tarot reading" 🤔

**Comment by DanM3rcurius** [comment]
> HTML or plain text should work. Probably best to see what format Occultish prefers.
> 
> Before we send out I think we should add categories and channels to the discord server that will speak to Tarot, NFT, and artist communities. Current discord is structured to support a DisCO and get work done. We also need to have content that gets people excited about art and Tarot that creates excitement and encourages interaction. Maybe something like "Get a live tarot reading" 🤔

Yes, I was thinking that the GitBook will be built out to function as a compendium also containing the lore of the metaverse.
A Live reading is a dope idea, but who has a setup (good camera, light, quiet surrounding, stable internet) to do this? We can invite to Tarot or NFT workshops (in collaboration with Space of Urgency maybe). 

For the newsletter itself, maybe we do one HTML and one plain text version?

**Comment by wlstylr** [comment]
> > HTML or plain text should work. Probably best to see what format Occultish prefers.
> > Before we send out I think we should add categories and channels to the discord server that will speak to Tarot, NFT, and artist communities. Current discord is structured to support a DisCO and get work done. We also need to have content that gets people excited about art and Tarot that creates excitement and encourages interaction. Maybe something like "Get a live tarot reading" 🤔
> 
> Yes, I was thinking that the GitBook will be built out to function as a compendium also containing the lore of the metaverse. A Live reading is a dope idea, but who has a setup (good camera, light, quiet surrounding, stable internet) to do this? We can invite to Tarot or NFT workshops (in collaboration with Space of Urgency maybe).
> 
> For the newsletter itself, maybe we do one HTML and one plain text version?

Gitbook is great for compendium!  But, we should still rework the discord server before sending out the newsletter.  We can figure out what is feasible when we plan the relaunch sprint.  The main point is that currently both or Discord and Github are structured for working and building.   I think it may make more sense to try to keep most of the "work" in Github, and then focus Discord more towards building community at the intersections of Tarot, art, and NFTs.

**Comment by DanM3rcurius** [comment]
Yes totally agree. How abozt just creating two new channels ("Reception" & "incubate") and setup new roles for the artist, so they only see those 2 channels. In case they want to dive deeper they can?
Also the shell channels are not really used, so I'm thinking of closing them. Thoughts?

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by wlstylr** [comment]
I like reception and incubate.  As we get closer to founders drop we will want our Discord structured in a way that encourages conversation and interaction.  We will have artists, tarot enthusiasts, and NFT enthusiasts all coming together.  Ideally, our Discord server can becomes a place that nurtures the community

**Comment by DanM3rcurius** [comment]
referencing https://github.com/SecretDecks/Website/issues/7

**Comment by DanM3rcurius** [comment]
@evolayak Could you please give this a final look and approval or request a call for feedback! Would be amazing if we can send this out in November :pray:

**Comment by evolayak** [comment]
Hey hey, changed the part abt ACO to DisCO so plz have a look!
